e2e/src/FiatConnectTransferOut.spec.js

```diff
@@ -5,7 +5,7 @@ import {
 import { MOCK_PROVIDER_BASE_URL, MOCK_PROVIDER_API_KEY } from 'react-native-dotenv'
 import { launchApp } from './utils/retries'
 
-describe('FiatConnect Transfer Out', () => {
+describe(':ios: FiatConnect Transfer Out', () => {
```

Comment:
would we just move this prefix to the KYC test when we get this working on Android?

Reply:
Yep, that would work.